### PR TITLE
[IMP] account,analytic,hr_timesheet: harmonize analytic search view

### PR DESCRIPTION
**Pull request by epictete**
The aim is to harmonize the analytic search view between `hr_timesheet` and `account` by creating an inheritance from `analytic`.
This allows the accountant to have access to all the fields and filters defined in other modules.

task-3463639

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_timesheet/report/hr_timesheet_report_view.xml [hunk 1]
```diff
@@ -81,11 +81,28 @@
         <record id="hr_timesheet_report_search" model="ir.ui.view">
             <field name="name">timesheets.analysis.report.search</field>
             <field name="model">timesheets.analysis.report</field>
-            <field name="inherit_id" ref="hr_timesheet.hr_timesheet_line_search_base"/>
```
Comment (william-andre) [inline addons/hr_timesheet/report/hr_timesheet_report_view.xml:84]: Shouldn't this still be inherited from `hr_timesheet.hr_timesheet_line_search`?

Comment (epictete) [reply addons/hr_timesheet/report/hr_timesheet_report_view.xml:84]: Nope, `hr_timesheet.hr_timesheet_line_search_base` is being removed with the `[REV]` commit :wink:

File: addons/sale_timesheet/report/timesheets_analysis_views.xml [hunk 2]
```diff
@@ -50,6 +50,41 @@
...
+                <filter string="Sales Order" name="groupby_sale_order" domain="[]"
+                    context="{'group_by': 'order_id'}" groups="sales_team.group_sale_salesman"/>
+                <filter string="Sales Order Item" name="groupby_sale_order_item" domain="[]"
+                    context="{'group_by': 'so_line'}" groups="sales_team.group_sale_salesman"/>
+                <filter string="Invoice" name="groupby_invoice" domain="[]"
+                    context="{'group_by': 'timesheet_invoice_id'}" groups="sales_team.group_sale_salesman"/>
+                <filter string="Billable Type" name="groupby_timesheet_invoice_type" domain="[]"
+                    context="{'group_by': 'timesheet_invoice_type'}" groups="sales_team.group_sale_salesman"/>
+            </xpath>
+        </field>
```
Comment (william-andre) [inline addons/sale_timesheet/report/timesheets_analysis_views.xml:85]: Just this?
```suggestion
        <field name="inherit_id" ref="sale_timesheet.timesheet_view_search"/>
        <field name="mode">primary</field>
        <field name="arch" type="xml">
            <!-- I think you need something in here for the view to be valid, so just a no-op action, to check if needed -->
            <xpath expr="//field[@name='task_id']" position="after"/>
        </field>
```

Comment (william-andre) [reply addons/sale_timesheet/report/timesheets_analysis_views.xml]: Looks like at least the order of loading needs to be different :sweat_smile: 
(I didn't test the suggestion)

**Review by william-andre — APPROVED** [review]
@robodoo r+

File: diff.txt [hunk 3]
```diff
@@ -0,0 +1,30 @@
+diff --git a/addons/account/tests/test_chart_template.py b/addons/account/tests/test_chart_template.py
```
Comment (aj-fuentes) [inline diff.txt:1]: Was this committed by mistake?

**Comment by william-andre** [comment]
@robodoo rebase-ff
@epictete can you remove that file?

File: diff.txt (hunk 3 above)
Comment (epictete) [reply diff.txt]: Sorry about that :sweat_smile:

**Comment by william-andre** [comment]
@robodoo r+
Didn't see that it didn't take the r- earlier :facepalm:

**Pull request closed without merging**